develop/tutorials/articles/230-web-experience-management/xx-liferay-fragments-generator/02-using-the-fragment-generator.markdown

```diff
@@ -0,0 +1,56 @@
+# Using the Fragment Generator
+
+After you setup the generator and create your first project by running
+`yo liferay-fragment` you will have the basic structure created, but there's
+still more that the generator can help you with.
```

Comment:
Maybe we could mention here that all existing task are defined inside `package.json > scripts`. They're just yeoman generator aliases

Reply:
Added info that "You can see all of the available tasks inside the `scripts` section in `package.json`."